uberfire-nio2-backport/uberfire-nio2-api/src/main/java/org/uberfire/java/nio/file/api/FileSystemUtils.java

```diff
@@ -18,6 +18,7 @@
 
 import java.util.Properties;
 
+import org.uberfire.commons.config.ServerType;
 import org.uberfire.java.nio.file.spi.FileSystemProvider;
 
 public class FileSystemUtils {
```

Comment:
@adrielparedes Need to update value of the constants to
`
public static final String CFG_KIE_CONTROLLER_OCP_ENABLED = "org.kie.server.controller.openshift.enabled";
`

Reply:
+1